js/theme.js

```diff
@@ -2355,10 +2355,10 @@
   // of the `<html>` and `<body>` rect bounds if horizontally scrollable
 
   function getDocumentRect(element) {
-    var _element$ownerDocumen;
+    var _element$ownerDocument;
```

Comment:
This comes from the Bootstrap source files and should be fixed there. The same goes for the other occurrences of `_element$ownerDocument`.